patches/natron_plugins-path.diff

```diff
@@ -27,7 +27,7 @@
  endif
  ifeq ($(OS),Linux)
 -  PLUGINPATH=/usr/OFX/Plugins
-+  PLUGINPATH=/OFX/Plugins
++  PLUGINPATH=/usr/lib/OFX/Plugins
```

Comment:
you don't normally get to write to /usr inside a flatpak, I haven't looked at this closely but are you sure you didn't mean `/app/lib/OFX/Plugins`?